radio/src/translations/de.h

```diff
@@ -342,16 +342,16 @@
 #define TR_MODELNAME                   "Modellname"
 #define TR_PHASENAME                   "Phase-Name"
 #define TR_MIXNAME                     "Mix-Name"
-#define TR_INPUTNAME                   TR("Input", "Input name")
-#define TR_EXPONAME                    TR("Name", "Line name")
+#define TR_INPUTNAME                   TR("Input", "Inputname")
+#define TR_EXPONAME                    TR("Name", "Zeilenname")
 #define TR_BITMAP                      "Modellfoto"
 #define TR_NO_PICTURE                  "kein Foto"
 #define TR_TIMER                       "Timer"
 #define TR_START                       "Start"
 #define TR_ELIMITS                     TR("Erw. Limit", "Erw. Wege auf 150%")
 #define TR_ETRIMS                      TR("Erw. Trims", "Erw. Trim  auf 100%")
 #define TR_TRIMINC                     TR("Trimschritt", "Trimmschritte")
-#define TR_DISPLAY_TRIMS               TR("Trimanzeige", "Trimwerte anzeigen")
+#define TR_DISPLAY_TRIMS               TR("Trimanzeige", "Trimmwerte anzeigen")
```

Comment:
Does it need to be changed to "Trimmanzeige" then as well?

Reply:
done